drivers/i2c/i2c_sam_twihs.c

```diff
@@ -10,17 +10,18 @@
  * Only I2C Master Mode with 7 bit addressing is currently supported.
  */
 
+#define SYS_LOG_DOMAIN "dev/i2c_sam_twihs"
```

Comment:
any reason this isn't in one of the cleanup patches?

Reply:
I had to rearrange the order due to `#include "i2c-priv.h"` which is new. `i2c-priv.h` defines `SYS_LOG_LEVEL`.

Reply:
ah, cool